modules/web-mercator/test/spec/fit-bounds.spec.js

```diff
@@ -3,7 +3,7 @@ import {fitBounds} from '@math.gl/web-mercator';
 import {WebMercatorViewport} from '@math.gl/web-mercator';
 import {toLowPrecision} from '../utils/test-utils';
 
-const FITBOUNDS_TEST_CASES = [
+const FITBOUNDS_TEST_CASES = /** @type {[import('@math.gl/web-mercator/fit-bounds').FitBoundsOptions, import('@math.gl/web-mercator/fit-bounds').Bounds][]} */ ([
```

Comment:
You need some sort of type annotation here to prevent TypeScript from widening the lat/lng bounds arrays to `number[][]` instead of `[[number, number], [number, number]]`.

If you'd rather declare the bounds type as `number[][]` then I don't think you need this.

(If the test files were TS, not JS, then the syntax would look a lot nicer!)